### ISSUE TITLE
XCode 11 - Scan doesn't run tests when "result_bundle: true" is used

### ISSUE THREAD

**Issue by sneha-je** [fastlane/fastlane#15561]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I've just updated Xcode to version 11.1 . When I run the tests using scan without using  "result_bundle: true" then the tests run fine but the tests fail to run when result_bundle is set to true with the error 
`xctest (19452) encountered an error (Early unexpected exit, operation never finished bootstrapping - no restart will be attempted. (Underlying error: Test runner exited before starting test execution.))`

##### Command executed

bundle exec fastlane tests

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### Environment

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>
